features/checkout/addressing_order/filling_shipping_and_billing_address_details_as_guest.feature

```diff
@@ -0,0 +1,27 @@
+@checkout
+Feature: Addressing an order
+    In order to address an order
+    As a Guest
+    I want to be able to fill addressing details
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+
+    @ui
+    Scenario: Address an order without different billing address
+        Given I have product "PHP T-Shirt" in the cart
+        And I am at the checkout addressing step
+        When I specify the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+        And I specify the email as "jon.snow@example.com"
```

Comment:
The e-mail field will be naturally above the address and the first thing customer should do is fill the address - please move this step.